styles.css

```diff
@@ -1,98 +1,134 @@
 .title {
-       font-family:Roboto;
+    font-family:Roboto;
 }
 
 .wrapper{
-        margin: auto;
-        width: 80%;
+    margin: auto;
+    width: 80%;
 }
 
 .header{
-       display: flex;
-       justify-content: space-between;
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 130px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/IrirniProject/pullRequest?prid=6241594)